library/common-library/templates/_agent-config.tpl

```diff
@@ -0,0 +1,17 @@
+{{/*
+This helper should return the defaults that all agents should have
+*/}}
+{{- define "common.agentConfig.defaults" -}}
+{{- if include "common.verboseLog" . }}
+verbose: 1
+{{- end }}
+{{- if (include "common.nrStaging" . ) }}
+staging: true
+{{- end }}
+{{- with include "common.proxy" . }}
+proxy: {{ . | quote }}
+{{- end }}
+{{- with include "common.fedramp.enabled" . }}
```

Comment:
the fedramp enable I believe returns just true/false

we should populate the correct endpoint for the agent

Reply:
I only found [this about fedramp](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#fedramp).

Which other variables do we need?

Reply:
oh, nothing, last time I checked I believe we needed to add manually each endpoint. Either I am wrong or they added it later on. Then it should be ok